status/controller_test.go

```diff
@@ -349,8 +702,9 @@ func GetMetric(name string, labels ...map[string]string) *prometheus.Metric {
 	return nil
 }
 
-func conditionLabels(t status.ConditionType, s metav1.ConditionStatus) map[string]string {
+func conditionLabels(k string, t status.ConditionType, s metav1.ConditionStatus) map[string]string {
```

Comment:
```suggestion
func conditionLabels(obj client.Object, t status.ConditionType, s metav1.ConditionStatus) map[string]string {
```

consider passing obj through this function over just the kind

Reply:
Updated to taken the `schema.GroupVersionKind` since that's all we need from the object